packages/@react-spectrum/menu/test/MenuTrigger.test.js

```diff
@@ -778,4 +779,133 @@ describe('MenuTrigger', function () {
     let menu2Item1 = within(menu2).getByText('Whiskey');
     expect(menu2Item1).toBeInTheDocument();
   });
+
+  describe('MenuTrigger trigger="longPress" open behavior', function () {
+    installPointerEvent();
+
+    const DEFAULT_THRESHOLD = 500;
+
+    it('should open the menu on longPress', function () {
+      const props = {onOpenChange, trigger: 'longPress'};
+      verifyMenuToggle(MenuTrigger, props, {}, (button, menu) => {
+        if (!menu) {
+          triggerLongPress(button);
+        } else {
+          triggerTouch(button);
+        }
+      });
+    });
+
+    it('should open the menu on Enter', function () {
+      const props = {onOpenChange, trigger: 'longPress'};
+      verifyMenuToggle(MenuTrigger, props, {}, (button, menu) => {
+        if (!menu) {
+          fireEvent.keyDown(button, {key: 'Enter'});
+          act(() => jest.advanceTimersByTime(DEFAULT_THRESHOLD));
+          fireEvent.keyUp(button, {key: 'Enter'});
+        } else {
+          triggerTouch(button);
+        }
+      });
+    });
+
+    it('should open the menu on Space', function () {
+      const props = {onOpenChange, trigger: 'longPress'};
+      verifyMenuToggle(MenuTrigger, props, {}, (button, menu) => {
+        if (!menu) {
+          fireEvent.keyDown(button, {key: ' '});
+          act(() => jest.advanceTimersByTime(DEFAULT_THRESHOLD));
+          fireEvent.keyUp(button, {key: ' '});
+        } else {
+          triggerTouch(button);
+        }
+      });
+    });
+
+    it('should open the menu on Alt+ArrowUp', function () {
+      const props = {onOpenChange, trigger: 'longPress'};
+      verifyMenuToggle(MenuTrigger, props, {}, (button, menu) => {
+        if (!menu) {
+          fireEvent.keyDown(button, {key: 'ArrowUp', altKey: true});
```

Comment:
should this be more granular to control, verse alt, verse command/windows key?

Reply:
Makes sense, i can add tests for windows key

Reply:
Hey @ktabors, I'm rereading the description of this issue. I'm wondering what's the expected behavior for command/windows key? Could you share more details? Thanks!